lib/system/gc_regions.nim

```diff
@@ -390,12 +390,14 @@ proc GC_disableMarkAndSweep() = discard
 proc GC_getStatistics(): string = return ""
 
 proc getOccupiedMem(): int =
+  ## Return the currently occupied memory as an integer.
```

Comment:
This procedure isn't exported (there is no asterisk by its name) so it doesn't need a doc comment :)

Reply:
Just because something isn't exported doesn't mean that it should not have a doc comment. My complaint is rather that this doc comment is very redundant, there is no additional information in it.